kotlin-audio/src/main/java/com/doublesymmetry/kotlinaudio/notification/NotificationManager.kt

```diff
@@ -35,56 +32,68 @@ class NotificationManager internal constructor(private val context: Context, pri
 
     private val buttons = mutableSetOf<NotificationButton?>()
 
-    private val channelId: String
-
     var notificationMetadata: NotificationMetadata? = null
         set(value) {
             field = value
-            internalManager?.invalidate()
+
+            scope.launch {
+                reload()
+            }
```

Comment:
Notification should be running on main thread, so I decided to call all functions in the `MainScope`. Didn't seem like it had any positive effect on this issue, but might be worth keeping it for safety.